brightlayer-ui/angular-component-library
# Feature/dev doc scaffold

There isn't a formal story for this made (yet).  But this changes the folder structure of the component-lib project to tightly couple the component lib & the component dev docs so they can be served together but built separately.  




#### To Test:


To test, I'd take a look at the folder structure to see how things are arranged.

Some commands to help see how things currently work: 

        "install:dependencies": "yarn && cd showcase && yarn", 
        "start": "ng serve demo -c development --open",    // Starts the dev docs using the local instance of the ng-comp lib/
        "start:npm": "ng serve demo -c npm --open",          // Starts the dev docs using the NPM instance of the ng-comp lib/
        "build:demo": "ng build demo -c production",         // Builds the dev docs 
        "build:lib": "bash scripts/build.sh",                            // Builds the comp lib
